pkg/utils/installer/install.go

```diff
@@ -366,7 +366,6 @@ func CreateDaemonSet(ctx context.Context, identity string, directCSIContainerIma
 				Name:  nodeDriverRegistrarContainerName,
 				Image: filepath.Join(registry, org, nodeDriverRegistrarContainerImage),
 				Args: []string{
-					fmt.Sprintf("--v=%d", logLevel),
```

Comment:
Removing this because, currently, the container fails to start with of the following error

```
Error: unknown flag: --v
```

We can address this in a separate PR

Reply:
This should not be removed. This will make us backwards incompatible